core/templates/core/footer.html

```diff
@@ -44,8 +82,8 @@
             </div>
         </div>
         <div class="contact">
-            Copyright &copy; 2015 Open Canada, All rights reserved<br>
-            Any trademarks or logos used herein are the property of their respective owners.
+            Copyright &copy; 2015 Open Canada, All rights reserved.<br>
```

Comment:
You can get the current year in a template using {% now "Y" %}, then we don't have to remember to update the copyright each year.